release.nix

```
@@ -0,0 +1,16 @@
+{ nixpkgs ? <nixpkgs>
+}:
+
+let
+  pkgs = import nixpkgs { config = {}; overlays = []; };
+in rec {
```

1. Why clear the overlays?
2. Why not make the import the default argument?

```suggestion
{ nixpkgs ? import <nixpkgs> { }
}:

rec {
```

---

I just generally think it's a good idea to clear overlays if I'm also going to build it with `nix-build`.  I've removed `release.nix` in latest commits.